invenio_github/ext.py

```diff
@@ -52,6 +52,19 @@ def release_api_class(self):
         assert issubclass(cls, GitHubRelease)
         return cls
 
+    @cached_property
+    def release_error_handlers(self):
+        """Github Release error handlers."""
+        error_handlers = current_app.config.get('GITHUB_ERROR_HANDLERS', {})
+        return [(import_string(error_cls), import_string(handler))
+               for error_cls, handler in error_handlers}]
+
+    @cached_property
+    def release_default_handler(self):
```

Comment:
There's no need to have the default handlers also as a property. They can be directly included in the task's `except` clause. If someone one's to override them, they can add a handler for `Exception` high up, in their `GITHUB_ERROR_HANDLERS`